File: InschrijvingenApp/ClientApp/app/components/models/Ouders.ts
import { Persoon } from "./persoon";
import { Adres } from "./Adres";

export class Ouders {
    constructor() {
        this.ouder1 = new Persoon();
        this.ouder2 = new Persoon();
        this.adres = new Adres();
        this.telefoon1 = "";
        this.telefoon2 = "";
        this.noodtelefoon = "";
        this.email1 = "";
        this.email2 = "";
    }

    public ouder1: Persoon;
    public ouder2: Persoon;
    public adres: Adres;
    public telefoon1: string;
    public telefoon2: string;
    public noodtelefoon: string;
    public email1: string;
    public email2: string;
}

File: InschrijvingenApp/Entities/Ouders.cs
using InschrijvenPietieterken.Entities;

namespace InschrijvingPietieterken.Entities
{
    public class Ouders : EntityBase
    {
        public Ouders()
        {
            Ouder1 = new Persoon();
            Ouder2 = new Persoon();
            Adres = new Adres();
        }

        public Persoon Ouder1 { get; set; }
        public Persoon Ouder2 { get; set; }
        public Adres Adres { get; set; }
        public string Telefoon1 { get; set; }
        public string Telefoon2 { get; set; }
        public string NoodTelefoon { get; set; }
        public string Email1 { get; set; }
        public string Email2 { get; set; }
    }
}